src/ae_nullptr_deref_stub_tests/recursive_func.c

```diff
@@ -1,19 +0,0 @@
-#include <stdio.h>
-
-extern void UNSAFE_LOAD(void *ptr);
```

Comment:
Why remove these cases? You could put them under a folder if they can't pass now.

Reply:
Ok, I will do that.

> Why remove these cases? You could put them under a folder if they can't pass now.